packages/popover/src/Popover/Popover.stories.jsx

```diff
@@ -334,3 +352,22 @@ export const RendersOnSpecificArea = {
     await new Promise((resolve) => setTimeout(resolve, 2000))
   },
 }
+
+export const DisableFloating = (args) => {
+  return (
+    <Flex m={200}>
+      <Popover
+        color='primary'
+        renderContent={LongTextContent}
+        placement='bottom'
+        ariaLabel='Default Popover'
+        width={330}
+        disableFloating={args.disableFloating}
+      >
+        <Button color='primary' mx={2}>
+          Open popover
+        </Button>
+      </Popover>
+    </Flex>
+  )
+}
```

Comment:
@LKeithLopez Mind adding an arg so the story will default to `disableFloating`? Here's an example https://github.com/priceline/design-system/blob/main/packages/carousel/src/Carousel.stories.jsx#L119